### DISCUSSION TITLE
[@types/papaparse] trasformHeader typescript error

### DISCUSSION THREAD

**Discussion by jhammersholt** [DefinitelyTyped/DefinitelyTyped#63469] [Issues with a @types package] [answered]
When using transformHeader with a remotefile i get this typescript error

`No overload matches this call.
  The last overload gave the following error.
    Argument of type '"https://docs.google.com/spreadsheets/d/1WLa7X8h3O0-aGKxeAlCL7bnN8-FhGd3t7pz2RCzSg8c/pub?gid=1558235212&output=csv"' is not assignable to parameter of type 'unique symbol'.ts(2769)`
    
Here is my parse call
` 
Papa.parse(remoteUrl, {
      download: true,
      header: true,
      transformHeader: (header, hIndex) => {
        console.group('header');
        console.log(hIndex, header);
        console.groupEnd();
        return hIndex;
      },
      complete: (results) => {
        console.log(results);
      },
    });
`

[a comment by an automated account is not shown]

**Comment by rainshen49** [comment, marked as answer]
I think your transformHeader needs to return a string. `hIndex` is a number. Try <code>return \`${hIndex}\`</code>
